2024-09-17 10:10:20,005 - root - INFO - Restored all variables from .//models/model.pt
2024-09-17 10:10:20,006 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-17 10:10:41,299 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: applying transform <monai.transforms.compose.Compose object at 0x7f6f93938970>
2024-09-17 10:10:41,308 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: applying transform <monai.transforms.compose.Compose object at 0x7f6f93938970>
2024-09-17 10:11:54,189 - root - INFO - Restored all variables from .//models/model.pt
2024-09-17 10:11:54,190 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-17 10:29:20,824 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: 
2024-09-17 11:17:46,142 - root - INFO - Restored all variables from .//models/model.pt
2024-09-17 11:17:46,143 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-17 11:23:59,252 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: 
2024-09-17 11:57:32,984 - root - INFO - Restored all variables from .//models/model.pt
2024-09-17 11:57:32,985 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-17 12:18:12,451 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:20:39.466
2024-09-17 12:18:12,451 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:20:39.466
2024-09-17 23:57:45,166 - root - INFO - Restored all variables from .//models/model.pt
2024-09-17 23:57:45,167 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-18 00:03:54,802 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: 
2024-09-18 00:04:15,932 - root - INFO - Restored all variables from .//models/model.pt
2024-09-18 00:04:15,933 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-18 00:25:41,063 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:21:25.130
2024-09-18 00:25:41,064 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:21:25.130
2024-09-18 01:39:59,924 - root - INFO - Restored all variables from .//models/model.pt
2024-09-18 01:39:59,925 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-18 02:01:29,626 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:21:29.701
2024-09-18 02:01:29,627 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:21:29.702
2024-09-18 02:17:41,695 - root - INFO - Restored all variables from .//models/model.pt
2024-09-18 02:17:41,696 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-18 02:22:30,173 - root - INFO - Restored all variables from .//models/model.pt
2024-09-18 02:22:30,174 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-18 02:38:37,685 - root - INFO - Restored all variables from .//models/model.pt
2024-09-18 02:38:37,686 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-18 02:58:26,638 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:19:48.951
2024-09-18 02:58:26,638 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:19:48.952
2024-09-18 02:59:15,562 - root - INFO - Restored all variables from .//models/model.pt
2024-09-18 02:59:15,563 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-18 03:18:59,556 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:19:43.993
2024-09-18 03:18:59,556 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:19:43.993
2024-09-18 04:02:07,998 - root - INFO - Restored all variables from .//models/model.pt
2024-09-18 04:02:07,999 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-18 04:03:36,596 - root - INFO - Restored all variables from .//models/model.pt
2024-09-18 04:03:36,597 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-18 04:24:01,707 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:20:25.110
2024-09-18 04:24:01,708 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:20:25.111
2024-09-18 21:59:13,426 - root - INFO - Restored all variables from .//models/model.pt
2024-09-18 21:59:13,427 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-18 22:07:15,918 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: 
2024-09-18 22:38:36,052 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-18 22:48:57,072 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:10:21.020
2024-09-18 22:48:57,072 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:10:21.020
2024-09-18 22:51:49,226 - root - INFO - Restored all variables from .//models/model.pt
2024-09-18 22:51:49,227 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-18 23:12:20,335 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:20:31.107
2024-09-18 23:12:20,335 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:20:31.108
2024-09-18 23:14:02,747 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-18 23:24:03,744 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:10:00.997
2024-09-18 23:24:03,745 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:10:00.997
2024-09-18 23:48:04,630 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-18 23:57:03,981 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: applying transform <monai.transforms.compose.Compose object at 0x7f19200203a0>
2024-09-18 23:57:03,985 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: applying transform <monai.transforms.compose.Compose object at 0x7f19200203a0>
2024-09-18 23:58:28,103 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-19 00:07:37,892 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: applying transform <monai.transforms.compose.Compose object at 0x7fe5e4504310>
2024-09-19 00:07:37,896 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: applying transform <monai.transforms.compose.Compose object at 0x7fe5e4504310>
2024-09-19 00:09:41,501 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-19 00:19:15,159 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: applying transform <monai.transforms.compose.Compose object at 0x7f2c5c3a43d0>
2024-09-19 00:19:15,163 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: applying transform <monai.transforms.compose.Compose object at 0x7f2c5c3a43d0>
2024-09-19 00:21:45,664 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-19 00:30:36,742 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:08:51.078
2024-09-19 00:30:36,742 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:08:51.078
2024-09-22 06:26:49,119 - root - INFO - Restored all variables from .//models/model.pt
2024-09-22 06:26:49,121 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-22 06:42:28,229 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:15:39.108
2024-09-22 06:42:28,229 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:15:39.108
2024-09-22 06:42:38,131 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-22 06:57:30,022 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:14:51.891
2024-09-22 06:57:30,023 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:14:51.891
2024-09-22 10:37:50,651 - root - INFO - Restored all variables from .//models/model.pt
2024-09-22 10:37:50,652 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-22 10:42:12,383 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:04:21.731
2024-09-22 10:42:12,383 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:04:21.731
2024-09-22 11:00:37,420 - root - INFO - Restored all variables from .//models/model.pt
2024-09-22 11:00:37,421 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-22 11:09:57,067 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:09:19.646
2024-09-22 11:09:57,068 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:09:19.647
2024-09-22 11:12:35,609 - root - INFO - Restored all variables from .//models/model.pt
2024-09-22 11:12:35,610 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-22 11:15:14,997 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:02:39.387
2024-09-22 11:15:14,998 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:02:39.387
2024-09-23 05:37:07,983 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 05:37:52,808 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 05:38:37,358 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 05:39:21,082 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 05:40:57,498 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 05:41:37,735 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 05:43:12,169 - root - INFO - Restored all variables from .//models/model.pt
2024-09-23 05:43:12,170 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 05:44:43,675 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: CUDA out of memory. Tried to allocate 192.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 127.69 MiB is free. Process 89410 has 23.56 GiB memory in use. Of the allocated memory 22.92 GiB is allocated by PyTorch, and 334.50 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-23 05:44:47,973 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: CUDA out of memory. Tried to allocate 192.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 127.69 MiB is free. Process 89410 has 23.56 GiB memory in use. Of the allocated memory 22.92 GiB is allocated by PyTorch, and 334.50 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-23 13:07:29,380 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:07:34,360 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: CUDA out of memory. Tried to allocate 192.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 17.69 MiB is free. Process 238230 has 23.67 GiB memory in use. Of the allocated memory 23.07 GiB is allocated by PyTorch, and 182.89 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-23 13:07:38,072 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: CUDA out of memory. Tried to allocate 192.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 17.69 MiB is free. Process 238230 has 23.67 GiB memory in use. Of the allocated memory 23.07 GiB is allocated by PyTorch, and 182.89 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-23 13:09:27,411 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:09:32,486 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: CUDA out of memory. Tried to allocate 192.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 17.69 MiB is free. Process 239365 has 23.67 GiB memory in use. Of the allocated memory 23.07 GiB is allocated by PyTorch, and 182.89 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-23 13:09:36,440 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: CUDA out of memory. Tried to allocate 192.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 17.69 MiB is free. Process 239365 has 23.67 GiB memory in use. Of the allocated memory 23.07 GiB is allocated by PyTorch, and 182.89 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-23 13:09:59,891 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:10:04,988 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: CUDA out of memory. Tried to allocate 192.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 17.69 MiB is free. Process 239894 has 23.67 GiB memory in use. Of the allocated memory 23.07 GiB is allocated by PyTorch, and 182.89 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-23 13:10:08,667 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: CUDA out of memory. Tried to allocate 192.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 17.69 MiB is free. Process 239894 has 23.67 GiB memory in use. Of the allocated memory 23.07 GiB is allocated by PyTorch, and 182.89 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-23 13:11:35,891 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:11:41,182 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: CUDA out of memory. Tried to allocate 192.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 17.69 MiB is free. Process 241016 has 23.67 GiB memory in use. Of the allocated memory 23.07 GiB is allocated by PyTorch, and 182.89 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-23 13:11:45,788 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: CUDA out of memory. Tried to allocate 192.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 17.69 MiB is free. Process 241016 has 23.67 GiB memory in use. Of the allocated memory 23.07 GiB is allocated by PyTorch, and 182.89 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-23 13:14:49,562 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:15:40,447 - root - INFO - Restored all variables from .//models/model.pt
2024-09-23 13:15:40,448 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:17:17,527 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: CUDA out of memory. Tried to allocate 192.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 165.69 MiB is free. Process 243790 has 23.52 GiB memory in use. Of the allocated memory 23.07 GiB is allocated by PyTorch, and 141.11 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-23 13:17:21,348 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: CUDA out of memory. Tried to allocate 192.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 165.69 MiB is free. Process 243790 has 23.52 GiB memory in use. Of the allocated memory 23.07 GiB is allocated by PyTorch, and 141.11 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-23 13:18:02,244 - root - INFO - Restored all variables from .//models/model.pt
2024-09-23 13:18:02,246 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:18:07,288 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: CUDA out of memory. Tried to allocate 192.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 139.69 MiB is free. Process 245552 has 23.55 GiB memory in use. Of the allocated memory 22.88 GiB is allocated by PyTorch, and 356.93 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-23 13:18:10,963 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: CUDA out of memory. Tried to allocate 192.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 139.69 MiB is free. Process 245552 has 23.55 GiB memory in use. Of the allocated memory 22.88 GiB is allocated by PyTorch, and 356.93 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-23 13:19:10,746 - root - INFO - Restored all variables from .//models/model.pt
2024-09-23 13:19:10,747 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:26:50,855 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: 
2024-09-23 13:27:28,272 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:27:54,602 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:28:06,719 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: CUDA out of memory. Tried to allocate 468.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 213.69 MiB is free. Process 254092 has 23.47 GiB memory in use. Of the allocated memory 22.50 GiB is allocated by PyTorch, and 567.55 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-23 13:28:06,723 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: CUDA out of memory. Tried to allocate 468.00 MiB. GPU 0 has a total capacity of 23.69 GiB of which 213.69 MiB is free. Process 254092 has 23.47 GiB memory in use. Of the allocated memory 22.50 GiB is allocated by PyTorch, and 567.55 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
2024-09-23 13:29:09,280 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:30:50,742 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: 
2024-09-23 13:31:02,555 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:31:08,908 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: 
2024-09-23 13:31:20,297 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:32:39,147 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: 
2024-09-23 13:32:53,231 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:34:22,374 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: applying transform <monai.transforms.compose.Compose object at 0x7f255c0f9520>
2024-09-23 13:34:22,380 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: applying transform <monai.transforms.compose.Compose object at 0x7f255c0f9520>
2024-09-23 13:35:26,341 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:36:55,911 - ignite.engine.engine.Vista3dEvaluator - ERROR - Current run is terminating due to exception: applying transform <monai.transforms.compose.Compose object at 0x7fb5b01b04c0>
2024-09-23 13:36:55,916 - ignite.engine.engine.Vista3dEvaluator - ERROR - Engine run is terminating due to exception: applying transform <monai.transforms.compose.Compose object at 0x7fb5b01b04c0>
2024-09-23 13:37:45,906 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:42:11,739 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:04:25.833
2024-09-23 13:42:11,740 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:04:25.833
2024-09-23 13:44:03,633 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:48:08,909 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:04:05.275
2024-09-23 13:48:08,909 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:04:05.276
2024-09-23 13:49:20,175 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run resuming from iteration 0, epoch 0 until 1 epochs
2024-09-23 13:52:11,697 - ignite.engine.engine.Vista3dEvaluator - INFO - Epoch[1] Complete. Time taken: 00:02:51.522
2024-09-23 13:52:11,698 - ignite.engine.engine.Vista3dEvaluator - INFO - Engine run complete. Time taken: 00:02:51.523
